.github/ISSUE_TEMPLATE/1_feature_template.md

```diff
@@ -10,15 +10,29 @@ What is the purpose, what´s the expected result.
 Please describe.
 -->
 
-## Impact
+## Impact/Dependencies
```

Comment:
I would not accept this, since slashes are always used without spacings.

> There are usually no spaces either before or after a slash. According to [New Hart's Rules: The Oxford Style Guide](https://en.wikipedia.org/wiki/New_Hart%27s_Rules:_The_Oxford_Style_Guide), a slash is usually written without spacing on either side when it connects single words, letters or symbols.

Source: https://en.wikipedia.org/wiki/Slash_(punctuation)